enumerables.rb

```diff
@@ -127,13 +127,12 @@ def my_none?(sth = nil)
           return false if item.match?(sth)
```

Comment:
- [ _recommended_] `return` statements must not be explicitly declared in your code. Ensure to remove all return statements in your code.

Reply:
Thank you for the review, will implement the recommendations requested.